Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/FormWithRedirect/PrimerHeadlessMainComponent.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexandra Lovin on 14.11.2023.
+//
+
```

Comment:
Is this something we can reuse outside of the `FormWithRedirect` scope? It seems quite generic, maybe it can be in the `Composable` directory to indicate it can be reused.